plugins/hydra_ax_sweeper/tests/config/defaults/default.yaml

```diff
@@ -0,0 +1,2 @@
+defaults:
+  - hydra/sweeper: ax
```

Comment:
Frankly, we should probably tell the user to activate this from the command line as the first choice.
and in that case, just don't use a config file and override the sweeper from the command line/overrides in the test.